### PR TITLE
Unittest gardening

### PR DESCRIPTION
**Pull request by gottesmm**
This PR contains a few fixes for unittests. I am just doing a drive by fix before I write some unittests for DiverseStack which I am going to rely on.

### CODE REVIEW

**Comment by gottesmm** [comment]
@swift-ci Please smoke test and merge

File: unittests/Basic/CMakeLists.txt [hunk 1]
```diff
@@ -6,23 +6,25 @@ handle_gyb_sources(
     ${SWIFT_HOST_VARIANT_ARCH})
 
 add_swift_unittest(SwiftBasicTests
-  ADTTests.cpp
+  ValueEnumeratorTest.cpp
+  OptionSetTest.cpp
+  RangeTest.cpp
```
Comment (erg) [inline unittests/Basic/CMakeLists.txt]: The rest of the tests are in alphabetical order.

**Comment by gottesmm** [comment]
@swift-ci Please smoke test and merge

**Comment by gottesmm** [comment]
@swift-ci Please smoke test and merge

**Comment by gottesmm** [comment]
@swift-ci Please smoke test and merge

**Comment by gottesmm** [comment]
@swift-ci Please smoke test and merge

**Pull request merged**